frontend/src/utilities/NavData.tsx

```diff
@@ -47,11 +47,29 @@ const useDSProjectsNav = (): NavDataItem[] =>
 
 const useDSPipelinesNav = (): NavDataItem[] => {
   const isAvailable = useIsAreaAvailable(SupportedArea.DS_PIPELINES).status;
+  const isExperimentsAvailable = useIsAreaAvailable(SupportedArea.PIPELINE_EXPERIMENTS).status;
 
   if (!isAvailable) {
     return [];
   }
 
+  // TODO temporary solution to switch between layout options
```

Comment:
nit: There are a few TODOs I'm seeing, and wondering if there should be JIRA links associated with them.

Reply:
hmm yeah. im not sure we have a jira for this. It might just be tracked by the artifacts and experiments epic. i can go in and make a jira to link here